Add inst0216 2

SaniHarouna-Mayer opened the pull request:
validated

sbillinge commented on db/institutions.yml:
```diff
@@ -1401,3 +1401,42 @@ zhejiangu:
+  aka:
+    - University of Manchester
+  name: The University of Manchester
+  city: Machester
```
typo

sbillinge commented on db/institutions.yml:
```diff
@@ -1401,3 +1401,42 @@ zhejiangu:
+  aka:
+    - McKinsey & Company, Geneva
+  name: McKinsey & Company, Inc. Switzerland
+  city: Genova
```
typo

sbillinge commented on db/institutions.yml:
```diff
@@ -1401,3 +1401,42 @@ zhejiangu:
+    flash:
+      name: Free electron laser FLASH
+  city: Hamburg
+  country: Deutschland
```
haha, nice, but I think that to be consistent we should put here Germany

SaniHarouna-Mayer replied:
ups.. I guess I was too tired yesterday hahah

sbillinge commented:
@SaniHarouna-Mayer looks like this needs a rebase now I am afraid

sbillinge commented:
needs another rebase I am afraid....cause I merged your 2016-2....

sbillinge commented:
oops @SaniHarouna-Mayer I am working with Berrak and I think we just conflicted you....

SaniHarouna-Mayer commented:
> oops @SaniHarouna-Mayer I am working with Berrak and I think we just conflicted you....

I rebased, added and committed, bit it doesn't push (or force push) and says everything is uptodate. Do I need to do more changes or is mergeable?

sbillinge commented:
ok, let's just close the PR.  I think somehow these edits made their way
into your other PR

On Mon, Feb 17, 2020 at 8:11 PM SaniHarouna-Mayer <notifications@github.com>
wrote:

> oops @SaniHarouna-Mayer <https://github.com/SaniHarouna-Mayer> I am
> working with Berrak and I think we just conflicted you....
>
> I rebased, added and committed, bit it doesn't push (or force push) and
> says everything is uptodate. Do I need to do more changes or is mergeable?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Billingegroup/rg-db-public/pull/200?email_source=notifications&email_token=ABAOWUN347OERP5K5LS5IJTRDMYURA5CNFSM4KWHZPY2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEMAGGVA#issuecomment-587227988>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABAOWUIJWU7444HI5VJSP73RDMYURANCNFSM4KWHZPYQ>
> .
>


-- 
Professor Simon Billinge
Columbia University

SaniHarouna-Mayer commented:
alright
